config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -41,29 +43,17 @@ presubmits:
       preset-dind-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
-        imagePullPolicy: Always
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191106-09649c7-master
         command:
         - runner.sh
         args:
         - ./hack/jenkins/test-dockerized.sh
-        env:
-        - name: KUBE_FORCE_VERIFY_CHECKS
```

Comment:
Why remove?

Reply:
these are for verify. they have nothing to do with integration.
I tested a run with mkpj